main/boost/APKBUILD

```diff
@@ -71,8 +71,9 @@ _libs="
 	wserialization
 	"
 for _lib in $_libs; do
-	case "$_options_carch " in # traing space is there to match last word
-	*"--without-${_lib} ") continue;;
+	# the trailing space in the below case is intentional so that we can match the last word
+	case "${_options_carch//$'\n'/ } " in
+		*"--without-${_lib} "*) continue ;;
```

Comment:
The only useful thing in this commit is the introduced `*` which is completely hidden in the useless changes. And commit message does not explain anything about why this change is needed.

However, the fact that builders has not choked over the bug this fixes, may indicate that we no longer need exclude things for s390x, so we can probably remove the `_options_carch` and the `case` conditional completely.